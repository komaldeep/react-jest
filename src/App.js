import React, { Component } from 'react';

import Button, {buttonSizes , buttonSkins}  from './atoms/Button/Button.js';
import InputText from './atoms/InputText/InputText';
import './App.css';

import WrapperLogin from './molecules/WrapperLogin/WrapperLogin'

class App extends Component {

  constructor (props) {
    super(props)
    this.state = {
      inputField:'',
    }
  }

  // inputFieldValue=(value)=>{
  //   this.setState({
  //     inputField: value
  //   })
  // }

  render() {
    return (
      <div>
        <div className="App-header App">
          <h2>Welcome to React</h2>
          <h3>Best Practise for writing react components (Atomic design)</h3>
        </div>
        <WrapperLogin />
      </div>
    );
  }
}

export default App;
