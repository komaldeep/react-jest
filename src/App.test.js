import React from 'react';
import renderer from 'react-test-renderer';

// jest.unmock('./App.css');
import App from './App';

it('renders without crashing', () => {
  const rendered = renderer.create(<App />).toJSON();
  expect(rendered).toMatchSnapshot();
});
