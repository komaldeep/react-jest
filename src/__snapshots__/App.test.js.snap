// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="App-header App"
  >
    <h2>
      Welcome to React
    </h2>
    <h3>
      Best Practise for writing react components (Atomic design)
    </h3>
  </div>
  <div>
    <label
      className="vertical"
    >
      Username
    </label>
    <div
      className="wrap"
    >
      <input
        className="input"
        disabled={false}
        id={undefined}
        maxLength={9999}
        name={undefined}
        onChange={[Function]}
        placeholder="Enter your name"
        type="text"
        value={undefined}
      />
    </div>
    <button
      className="btn small secondary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Submit
    </button>
  </div>
</div>
`;
